docs/flashbots-core/searchers/quick-start.md

```diff
@@ -11,45 +11,24 @@ This quickstart guide contains all the information necessary to get up and runni
 3. Ethereum Dapps with advanced use cases like account abstraction or gasless transactions (for example, tornado.cash and mistX)
 
-### How to send your first private transaction
+### How does Flashbots work for searchers?
+Flashbots connects searchers directly to miners and allows them to avoid public mempools. Searchers with transactions they would like to send miners first craft what we call "bundles" and send these to Flashbots' MEV-Relay. MEV-Relay is a gateway that Flashbots runs today which simulates searchers' bundles, and if there are no errors then forwards them on to miners. Miners then receive bundles and include them in blocks if it is profitable for them to do so.
 
-// todo //
+Getting onboarded to Flashbots is easy for searchers; you simply need to update how you send transactions.
 
 ### How to send your first Flashbots bundle
-To access the Flashbots network, you must craft a valid Flashbots bundle and send it to the `eth_sendBundle` RPC endpoint at `relay.flashbots.net`.
-
-The `eth_sendBundle` RPC has the following payload format:
-
-```json
-{
-  "jsonrpc": "2.0",
-  "id": 1,
-  "method": "eth_sendBundle",
-  "params": [
-    {
-      signedTxs,         // Array[String], A list of signed transactions to execute in an atomic bundle
-      blockNumber,       // String, a hex encoded block number for which this bundle is valid on
-      minTimestamp,      // (Optional) Number, the minimum timestamp for which this bundle is valid, in seconds since the unix epoch
-      maxTimestamp,      // (Optional) Number, the maximum timestamp for which this bundle is valid, in seconds since the unix epoch
-      revertingTxHashes, // (Optional) Array[String], A list of tx hashes that are allowed to revert 
-    }
-  ]
-}
-```
-
-To authenticate your request, the relay requires you sign the payload and include the signed payload in the `X-Flashbots-Signature` header of your request.
-
-```curl
-curl -X POST -H "X-Flashbots-Signature: 0x1234:0xabcd" --data '{"jsonrpc":"2.0","method":"eth_sendBundle","params":[{see above}],"id":1}' https://relay.flashbots.net
-```
+To access the Flashbots network you will need three things:
+1. A private key that Flashbots can use to identify you
+2. A way to interact with the Flashbots network
+3. A "bundle" for your transactions
 
-Any valid Ethereum key can be used to sign the payload. The Ethereum address associated with this key will be used by the relay to keep track of your requests over time and provide user statistics. You can change the key you use at any time.
+Let's begin with the private key Flashbots uses for identity. When you send bundles to Flashbots you will sign them with a private key so that we can establish identity for searchers and establish reputation for them over time. This private key **does not** store funds and is **not** the primary private key you use for executing transactions. Again, it is only used for identity, and it can be any private key.
```

Comment:
might want to explain we don't actually link to real-world identity (pseudonymous) and that the reputation is currently not an active feature of the system?